Sources/Common/Core/CellArray/index.js

```diff
@@ -72,14 +72,32 @@ function vtkCellArray(publicAPI, model) {
     model.cellSizes = undefined;
   };
 
-  /**
-   * Returns the point indexes at the given location as a subarray.
-   */
   publicAPI.getCell = (loc) => {
     let cellLoc = loc;
     const numberOfPoints = model.values[cellLoc++];
     return model.values.subarray(cellLoc, cellLoc + numberOfPoints);
   };
+
+  const superInitialize = publicAPI.initialize;
+  publicAPI.initialize = () => {
+    superInitialize();
+    model.numberOfCells = undefined;
+    model.cellSizes = undefined;
```

Comment:
What's the purpose for setting these two properties to `undefined`?

Reply:
It is to make sure that `insertNextCell` works properly after `initialize`.

Reply:
To document in index.d.js.

Though I do not know whether it is better to `undefined` those or delete them from `model`.